fluentd/fluent.conf

```diff
@@ -111,9 +132,11 @@
   pos_file /var/log/node.log.pos
   tag system.var.log
   format multiline
-  format_firstline /^[A-Z][a-z]{2} [ |0-3]+[0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
-  format1 /^(?<time>[^ ]* [^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
+  format_firstline /^[A-Z][a-z]{2} [ |0-3][0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
```

Comment:
We're making some assumptions about the format here. For instance that the month will be 3 letters - not sure that's true in all locales. Also I'm pretty sure timestamp can take quite a variety of formats. But I guess for now it's enough to cover the default. Still, a couple of nits:

1. Whenever I match space in a regex I prefer \s+ so I don't have to think about how much or what kind of whitespace to match. The only problem is if this ends up matching newlines, but I wouldn't think in this context that would happen.
2. Leap seconds look like `:60` :)

So might I propose this:
```
# Begin possible multiline match: "Mmm DD HH:MM:SS "
format_firstline /^[A-Z][a-z]{2}\s+[0-3][0-9]\s+[0-2][0-9]:[0-5][0-9]:[0-6][0-9]\s/
```